settlement/src/execution.rs

```diff
@@ -23,16 +23,16 @@ pub enum ExecutionMessage {
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
 pub struct TransferFungibleToken {
-    pub token_address: String,
+    pub token_address: HexSerializedVec,
     pub amount: u128,
-    pub receiver_address: String,
+    pub receiver_address: HexSerializedVec,
 }
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
 pub struct TransferNonFungibleToken {
-    pub collection_address: String,
-    pub token_index: String,
-    pub receiver_address: String,
+    pub collection_address: HexSerializedVec,
+    pub token_index: HexSerializedVec,
```

Comment:
Why is this field HexSeriazliedVec? Uint128 would be enough.

Reply:
Or string as before.

Reply:
As I explained before, `execution` module must not be specific to EVM.